src/chains/tests/test_signals.py

```diff
@@ -1,88 +1,101 @@
 import responses
 from django.test import TestCase, override_settings
+from faker import Faker
 
 from ..models import Feature, Wallet
-from ..tests.factories import ChainFactory, GasPriceFactory
+from .factories import ChainFactory, FeatureFactory, GasPriceFactory, WalletFactory
 
+fake = Faker()
+Faker.seed(0)
 
-@override_settings(
-    FF_HOOK_EVENTS=False,
-    CGW_URL="http://127.0.0.1",
-    CGW_FLUSH_TOKEN="example-token",
-)
-class ChainNetworkHookTestCase(TestCase):
+
+class ChainNetworkHookTestCaseSetupCheck(TestCase):
+    @responses.activate
+    @override_settings(CGW_URL=None, CGW_FLUSH_TOKEN="example-token")
+    def test_no_cgw_call_with_no_url(self) -> None:
+        ChainFactory.create()
+
+        assert len(responses.calls) == 0
+
+    @responses.activate
+    @override_settings(CGW_URL="http://127.0.0.1", CGW_FLUSH_TOKEN=None)
+    def test_no_cgw_call_with_no_token(self) -> None:
+        ChainFactory.create()
+
+        assert len(responses.calls) == 0
+
+
+@override_settings(CGW_URL="http://127.0.0.1", CGW_FLUSH_TOKEN="example-token")
+class ChainNetworkHookWithFFHookEventsTestCase(TestCase):
```

Comment:
Nit: let's remove "FF" from the name as we are no longer testing that (from my understanding).